### PR TITLE
Enables Octokit pagination for larger datasets

### PR DESCRIPTION
**Pull request by petems**
I was having issues with loading some users with a large amount of PRs.

Enabling pagination is an easy win :+1:

### CODE REVIEW

**Comment by leomilrib** [comment]
Hello there!

This is very nice! I was wishing to add this feature but ended up focusing in other things.

I'm gonna check what is breaking in the CI and see how this behaves with multiple organizations.

Thanks for using and suggesting! Oh, and sorry for the delay, I was out of town for some days. hehe

**Comment by leomilrib** [comment]
Okay, so it was my fault on the CI, wrong Ruby version :sweat_smile:  Fixed and passing!

File: lib/helpers.rb [hunk 1]
```diff
@@ -34,8 +34,10 @@ def error_and_back(error)
   def set_client
     begin
       if settings.development?
+        Octokit.auto_paginate = true
         client = Octokit::Client.new(:netrc => true)
       else
+        Octokit.auto_paginate = true
```
Comment (leomilrib) [inline lib/helpers.rb]: Okay, this lines are duplicated now, can you put it [here](https://github.com/leomilrib/junkie/blob/master/app.rb#L18) with the other _Octokit_ settings?

**Comment by leomilrib** [comment]
After changing the settings location we are good to go! :grin: :+1: 

And thanks again!

File: lib/helpers.rb (hunk 1 above)
Comment (petems) [inline lib/helpers.rb]: Fixed :+1:

**Pull request merged**